packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -112,9 +112,10 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
           this.updateStickyRefHeights();
         } else {
           // Else if mutation occurs in scrollable region, then find sticky it belongs to and force update
-          const stickyList = Array.from(this._stickies).filter((sticky) => {
-            if (this.root) {
-              return this.root.contains(mutation[0].target);
+          let stickyList: Sticky[] = [];
```

Comment:
/home/travis/build/OfficeDev/office-ui-fabric-react/packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx:115:15
ERROR: 115:15  prefer-const          Identifier 'stickyList' is never reassigned; use 'const' instead of 'let'.
ERROR: 326:13  prefer-const          Identifier 'stickyList' is never reassigned; use 'const' instead of 'let'.
ERROR: 329:33  no-shadowed-variable  Shadowed name: 'sticky'
ERROR: 331:36  semicolon             Missing semicolon
ERROR: 333:36  semicolon             Missing semicolon

Reply:
Install tslint extension in vscode for great justice.